server/.env

```diff
@@ -1 +1 @@
-TEAM_MEMBERS="Aecio, Bonnie, Shums"
+TEAM_MEMBERS="Aecio, Bonnie, Shums Kassm"
```

Comment:
It looks like there's a spelling mistake here. Otherwise looks good to me!

Reply:
Lol, learn your own name Shums